### DISCUSSION TITLE
ERROR in ../node_modules/@azure/communication-call-automation/dist-esm/src/callRecording.js 6:0-25

### DISCUSSION THREAD

**Discussion by developerbaba443** [community/community#128013] [Programming Help]
### Select Topic Area

Question

### Body

import { CallAutomationClient } from '@azure/communication-call-automation';
const callAutomationClient = new CallAutomationClient.CallAutomationClient("endpoint=.........);

Module not found: Error: Can't resolve 'fs'

**Comment by Rishi-Sudhakar** [comment]
If you're working in a browser environment (e.g., using Webpack), some Node.js-specific modules like 'fs' may not be available or may need to be polyfilled.

**Reply by IhfazAlam** [reply]
Hello

**Reply by IhfazAlam** [reply]
Rishi-Sudhakar

**Reply by IhfazAlam** [reply]
Where are you from? Rishi-Sudhakar
